<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTask</class>
 <widget class="QDialog" name="AddTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="addTaskMessageLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>300</width>
     <height>70</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <italic>true</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Add Task Details</string>
   </property>
  </widget>
  <widget class="QLabel" name="DescriptionLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>150</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QLabel" name="taskNameLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>150</width>
     <height>70</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Task Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="DateLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>420</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Date:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="datepicker">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>425</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="AddTaskButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>490</y>
     <width>150</width>
     <height>70</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add Task</string>
   </property>
  </widget>
  <widget class="QLabel" name="connectionMessageLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>550</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>[+]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TaskNameLineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>400</width>
     <height>60</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="DescriptionLineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>400</width>
     <height>150</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>150</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="successfullMessageLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
